cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt

```diff
@@ -160,40 +169,102 @@ EOS
+      "uaenorth":"AE North",
+      "southafricawest":"ZA West",
+      "francesouth":"FR South",
+      "westcentralus":"US West Central",
+      "koreacentral":"KR Central",
+      "westeurope":"EU West",
+      "southafricanorth":"ZA North",
+      "southeastasia":"AP Southeast",
+      "centralindia":"IN Central",
+      "switzerlandwest":"CH West",
+      "norwayeast":"NO East",
+      "germanywestcentral":"DE West Central",
+      "westus2":"US West 2",
+      "australiacentral":"AU Central",
+      "italynorth":"IT North",
+      "centralus":"US Central",
+      "germanynorth":"DE North",
+      "brazilsoutheast":"BR Southeast",
+      "southindia":"IN South",
+      "swedencentral":"SE Central",
+      "francecentral":"FR Central",
+      "australiasoutheast":"AU Southeast",
+      "northeurope":"EU North",
+      "koreasouth":"KR South",
+      "usgovtexas":"US Gov TX",
+      "polandcentral":"PL Central",
+      "japaneast":"JA East",
+      "westindia":"IN West",
+      "japanwest":"JA West",
+      "westus":"US West",
+      "jioindiawest":"IN West Jio",
+      "northcentralus":"US North Central",
+      "southcentralus":"US South Central",
+      "eastasia":"AP East",
+      "jioindiacentral":"IN Central Jio",
+      "australiacentral2":"AU Central 2",
+      "canadaeast":"CA East",
+      "eastus":"US East",
+      "uaecentral":"AE Central",
+      "norwaywest":"NO West",
+      "canadacentral":"CA Central",
+      "uksouth":"UK South",
+      "qatarcentral":"QA Central",
+      "swedensouth":"SE South",
+      "brazilsouth":"BR South",
+      "australiaeast":"AU East",
+      "switzerlandnorth":"CH North",
+      "usgovarizona":"US Gov AZ",
+      "westus3":"US West 3"
+    };
+
+    function getPriceInfo(location, vmSize, osType, pricingData) {
+      if (!pricingData[location] || !pricingData[location][vmSize] || !pricingData[location][vmSize][osType]) {
+        return null;
+      }
+
+      var pricePerUnit = pricingData[location][vmSize][osType].pricePerUnit
+      var pricePerUnitAHUB =  pricingData[location][vmSize][osType].pricePerUnitAHUB
+
+      if (pricePerUnit && pricePerUnitAHUB) {
+        return pricePerUnit - pricePerUnitAHUB;
       } else {
-        if (instance.osType === "Windows" ){
-          var resourceGroup = instance.id.split('/')[4]
-          var tags = JSON.stringify(instance["tags"]);
-          var ahub = "N/A"
-          if (instance.licenseType === "Windows_Server" || instance.licenseType === "Windows_Client") {
-            ahub = "Yes"
-          } else {
-            ahub = "No"
-          }
+        return null;
+      }
+    }
+
+    _.each(azure_instances, function(value) {
+      var regionMapping = region_table[value.location];
+      var priceDifference = getPriceInfo(regionMapping, value.vmSize, value.osType, pricing);
+        if (value.osType == "Windows" && priceDifference != null && value.licenseType == null){
+          monthlySavings = ((priceDifference * 24) * 30 )
```

Comment:
To match parity with some changes we are making to our savings policies, the math here should be slightly different. Replace 30 with 365.25 / 12.

Reasoning: 365.25 is the average number of days in a year (due to a leap year every 4 years), so this gives us a much closer approximation of what the average monthly cost would be.